be/src/storage/tablet_manager.cpp

```diff
@@ -394,8 +394,11 @@ Status TabletManager::drop_tablet(TTabletId tablet_id, TabletDropFlag flag) {
 
         LOG(INFO) << "Start to drop tablet " << tablet_id;
         dropped_tablet = it->second;
-        tablet_map.erase(it);
-        _remove_tablet_from_partition(*dropped_tablet);
+        dropped_tablet->set_is_dropping(true);
```

Comment:
Without acquiring the Tablet lock, can the scope of this lock completely solve the concurrency problem?